lib/admin/WP_Auth0_Admin_Basic.php

```diff
@@ -229,7 +229,7 @@ public function render_basic_description() {
 	public function auth0_delete_cache_transient() {
 		if ( ! is_admin() ) return;
 
-		WP_Auth0_ErrorManager::insert_auth0_error( 'WP_Auth0_Admin_Basic::delete_cache_transient', 'deleting cache transient' );
+		WP_Auth0_ErrorManager::insert_auth0_error( __METHOD__, 'deleting cache transient' );
```

Comment:
I must be missing something here, it seems like it will just log this error every time and then it calls the method to delete cache?

Reply:
It looks to me like it's there as a message rather than an error. I've never had a use for it (or seen it in the logs) but I'm guessing it's there for a reason.

Reply:
Well should be a general log vs error. However that can be a story for another day.